spock-core/src/main/java/spock/lang/Tag.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package spock.lang;
+
+import org.spockframework.runtime.extension.ExtensionAnnotation;
+import org.spockframework.runtime.extension.builtin.TagExtension;
+import org.spockframework.util.Beta;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Indicates that a feature method or specification belongs to a certain user-defined category.
+ *
+ * @since 2.2
+ * @author Alexander Kriegisch
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.TYPE, ElementType.METHOD})
+@ExtensionAnnotation(TagExtension.class)
+@Repeatable(Tag.Container.class)
+public @interface Tag {
+  /**
+   * @return the user-defined categories which the annotated element belongs to
+   */
+  String[] value();
```

Comment:
🤔 This should be just a singluar String, as the annotation is already repeatable.
I know that `@Issue` also has an array here, but that is because the repeated annotation support for extensions was added later and to keep backwards compatibility.